rust-libs/db/build.rs

```
@@ -21,13 +12,28 @@ fn patch_abseil_source() {
         return;
     }
 
-    let status = process::Command::new("sed")
+    process::Command::new("sed")
```

Is it me or are you ignoring errors here?

---

It *is* me, you're using `expect`, so if the file doesn't exist in *both* places, the build will fail. No good.